kafka/pom.xml

```diff
@@ -85,6 +85,18 @@
             <version>${test-containers.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
+            <version>2.10.1</version>
```

Comment:
Please add the jackson version to the main `pom.xml` and reference it in both dependencies.